includes/class-db.php

```diff
@@ -94,20 +94,23 @@ public static function update_options_key( $old, $new ) {
 	public static function migrate_to_1_0_0() {
 		// 1. rename comment meta
 		self::update_commentmeta_key( 'semantic_linkbacks_avatar', 'avatar' );
-		self::update_commentmeta_key( 'semantic_linkbacks_author_url', 'webmention_author_url' );
-		self::update_commentmeta_key( 'semantic_linkbacks_canonical', 'webmention_canonical_url' );
```

Comment:
This should be self::update_commentmeta_key( 'semantic_linkbacks_canonical', 'url' );

Reply:
done